README.md

```diff
@@ -101,6 +101,21 @@ but the short of it is below.
 - If you're not sure about it or want to chat, reach out on our [Discord](https://discord.gg/GravUqY) 
 - If you want to write some user docs 🎉 head over to the [stargate/docs](https://github.com/stargate/docs) repo, Pull Requests accepted!
 
+## Supported Versions and Branching Strategy
+
+The Stargate project maintains support for the current major version number and one major version number previous. We mark the previous major version number as deprecated in order to encourage usage of the latest version. We anticipate supporting a major version number release N for one year after the subsequent major version number release (N+1), or until the major version number release (N+2), whichever comes first. For example, v1 will be supported until when Stargate v3 is released, but no later than October 2023 (1 year after release date of Stargate v2). We anticipate producing a new major version number release every 6-12 months.
```

Comment:
`We anticipate producing a new major version number release every 6-12 months.` 

Don't think this is true?

Reply:
I made this update based on a conversation with Mark, but I can remove the statement.

Reply:
Actually I am going to merge as this is consistent with what was checked into `v1` branch. It is a statement of intent rather than a commitment and does not cause any harm.